style.css

```diff
@@ -442,3 +446,177 @@ ul.social-media-icons {
 .contact_container button:hover {
   cursor: pointer;
 }
+
+@media (min-width: 768px) {
+  .humberger ul {
+    display: flex;
+    padding: 0 8px;
+  }
+
+  .humberger ul li {
+    padding: 0 8px;
+  }
+
+  .humberger img {
+    display: none;
+  }
+
+  .head {
+    width: 90%;
+    margin: 0 auto;
+  }
+
+  .navbar__logo .logo {
+    font-size: 1.7rem;
+    margin: 10px 0 0 20px;
+  }
+
+  .works-card-img {
+    width: 500px;
+    height: 350px;
+  }
+
+  /* .head {
+    background-image: url(./assets/header_bg.png);
+    background-origin: border-box;
+    background-size: contain;
+    height: 510px;
+    background-position: center;
+    background-repeat: no-repeat;
+  } */
```

Comment:
- [ ] It is a good practice to remove all commented code in order to keep your work neat.

Reply:
* Removed all commented code.